RNFastCrypto.podspec

```
@@ -0,0 +1,21 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "RNFastCrypto"
+  s.version      = package['version']
+  s.summary      = "RNFastCrypto"
+  s.description  = package['description']
+  s.homepage     = "https://github.com/ExodusMovement/react-native-fast-crypto.git"
+  s.license      = "MIT"
+  s.author             = { "Paul Puey" => "paul@airbitz.co" }
+  s.platform     = :ios, "7.0"
+  s.source       = { :git => "https://github.com/ExodusMovement/react-native-fast-crypto.git", :tag => "v#{s.version}" }
+  s.source_files  = "RNFastCrypto/**/*.{h,m}"
```

If we need proper podspec we might want [this](https://guides.cocoapods.org/syntax/podspec.html#vendored_libraries) as well, but I don't think it matters.

```suggestion
  s.source_files  = "RNFastCrypto/**/*.{h,m}"
  s.vendored_libraries = 'RNFastCrypto/Libraries/libboost_regex.a', 'RNFastCrypto/Libraries/libboost_system.a', 'RNFastCrypto/Libraries/libboost_thread.a', 'RNFastCrypto/Libraries/libmymonerocorecpp.a', 'RNFastCrypto/Libraries/libnativecrypto.a', 
```